docker-compose.yml

```diff
@@ -333,11 +321,7 @@ services:
       - postgres
       - kafka
       - data-broker
-    depends_on:
-      postgres:
-        condition: service_healthy
```

Comment:
I`d appreciate a review about this modification. The double use of depends_on was causing errors, then i removed the condition.